translations/messages.fr.xlf

```diff
@@ -2326,17 +2326,17 @@
         <source>Moving the spider</source>
         <target>Déplacer l'araignée</target>
       </trans-unit>
-      <trans-unit id="7345241358894299630" datatype="html">
-        <source>Finally, the spider can move exactly 3 spaces at a time. It is not allowed to go twice through the same space in a move. As the spider crawls around other pieces, it can only move around pieces that it is in direct contact with. To do a spider move, select a spider and click on the three spaces that compose the move.&lt;br/&gt;&lt;br/&gt;Again, as Dark, your queen bee is stuck but you can free it by moving your spider, do it!</source>
-        <target>Finalement, l'araignée peut se déplacer d'exactement 3 cases. Elle n'est pas autorisée à passer deux fois par la même case au lors d'un tour. L'araignée se déplace le long des autres pièces : elle ne peut donc que se déplacer autour des pièces avec lesquelles elle est en contact direct. Pour déplacer une araignée, sélectionnez-en une et cliquer sur les trois cases qui composent son mouvement.&lt;br/&gt;&lt;br/&gt;En tant que Foncé, votre reine est à nouveau bloquée mais vous pouvez la libérer en déplaçant votre araignée, faites-le !</target>
+      <trans-unit id="1211369340024090138" datatype="html">
+        <source>Finally, the spider can move exactly 3 spaces at a time. It is not allowed to go twice through the same space in a move. As the spider crawls around other pieces, it can only move around pieces that it is in direct contact with. This means that the spider is not allowed to temporarily split the hive: here, Dark&apos;s spider cannot go on the left as its first move for example. To do a spider move, select a spider and click on the three spaces that compose the move.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, move your spider!</source>
```

Comment:
pieces that are in direction with it?
Le "par exemple" en fin de phrase c't'un peu bizarre non ?

Reply:
direction ?! :D (j'ai changé comme à l'autre endroit)
J'ai splitté la phrase au : et préfixé par le par exemple